examples/gno.land/p/demo/avl/node.gno

```diff
@@ -321,43 +344,36 @@ func (node *Node) TraverseInRange(start, end string, ascending bool, leavesOnly
 	}
 
 	// Run callback per inner/leaf node.
-	stop := false
 	if (!node.IsLeaf() && !leavesOnly) ||
 		(node.IsLeaf() && startOrAfter && beforeEnd) {
-		stop = cb(node)
-		if stop {
-			return stop
+		if cb(node) {
+			return true
 		}
 	}
+
 	if node.IsLeaf() {
-		return stop
+		return false
 	}
 
 	if ascending {
 		// check lower nodes, then higher
-		if afterStart {
-			stop = node.getLeftNode().TraverseInRange(start, end, ascending, leavesOnly, cb)
-		}
-		if stop {
-			return stop
+		if afterStart && node.getLeftNode().TraverseInRange(start, end, ascending, leavesOnly, cb) {
```

Comment:
can we keep the `stop` for sake of readability?

right now it take too much of my brain cycles to understand the conditions.

Reply:
reverted https://github.com/gnolang/gno/pull/1881/commits/1a6845c25ac77459ba7588b676016dbe79bbff70